grounding_map.csv

```diff
@@ -2659,3 +2659,4 @@ Rp1,HGNC,RP1,,,,
 RP-1,HGNC,RP1,,,,
 Psk2,UP,Q08217,,,,
 PSK2,UP,Q08217,,,,
+liposomes,MESH,D008081,,,,

```

Comment:
I don't think this is relevant for the PR... just pointing it out to make sure it doesn't get lost or forgotten :)

Reply:
Yes, just a local commit that ended up on the PR, but I was going to push it anyway so let's leave it